Nimb3s.Automaton.DataAccess/SqlDataAccess.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Dapper;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Text;
+
+namespace Nimb3s.Automaton.DataAccess
+{
+    public class SqlDataAccess
+    {
+        public List<T> GetData<T, U>(string sqlStatement, U parameter, string connectionString)
+        {
+            using (IDbConnection connection = new SqlConnection(connectionString))
+            {
+                List<T> rows = connection.Query<T>(sqlStatement, parameter).ToList();
+                return rows;
+            }
+        }
+    }
+}
```

Comment:
after the code is moved to the [JobStatusRepository](https://github.com/nimb3s/automaton/blob/main/src/Nimb3s.Automaton.Core.Repositories.Sql/JobStatusRepository.cs) (see comments on `Nimb3s.Automaton.DataAccess/JobData.cs`) we can remove this library from the solution and from disk. all db logic will go into the [`Nimb3s.Automaton.Core.Repositories.Sql`](https://github.com/nimb3s/automaton/tree/main/src/Nimb3s.Automaton.Core.Repositories.Sql) library